feat(lb): add troubleshooting (#4591)
scaleway/docs-content: faq/loadbalancer.mdx

By default, each [public](/load-balancer/concepts/#accessibility) Load Balancer is created automatically with a flexible IPv4 address. This is a public IP that can be held in your account even after you delete your Load Balancer. You can optionally also add an IPv6 address.

> **This IP address is fixed and does not risk changing once attached to your Load Balancer**.

Reviewer: It's not fixed. It's flexible :) 
You can attach it to any other LB at any time.

```suggestion
**This IP address is reserved and does not risk changing as long as it remains attached to your Load Balancer**.
```
